client/my-sites/themes/theme-showcase.jsx

```diff
@@ -48,15 +49,41 @@ const ThemeShowcase = React.createClass( {
 		this.setState( { showPreview: ! this.state.showPreview, previewingTheme: theme } );
 	},
 
-	onPreviewButtonClick( theme ) {
-		const { defaultOption } = this.props;
+	onPrimaryPreviewButtonClick( theme ) {
+		const option = this.getPrimaryOption();
 		this.setState( { showPreview: false }, () => {
-			defaultOption.action && defaultOption.action( theme );
+			option.action && option.action( theme );
 		} );
 	},
 
+	onSecondaryPreviewButtonClick( theme ) {
```

Comment:
These two `...buttonClick` funcs could be combined into one by passing the option in, but again it is fine and straightforward as is.